### PR TITLE
Add step-62 to the tutorials list

### PR DESCRIPTION
**Pull request by dangars**
Following up on the conversation https://github.com/dealii/dealii/pull/6747#issuecomment-488599406 I added the tutorial step-62 to the Tutorials list.

I added a couple of items to the _"Advanced techniques"_ section: _"Parallelization via MUMPS and MPI"_ and _"HDF5 and Python"_

### CODE REVIEW

File: doc/doxygen/tutorial/tutorial.h.in [hunk 1]
```diff
@@ -445,6 +445,13 @@
  *       on non-matching grids.
  *       </td></tr>
  *
+ *   <tr valign="top">
+ *       <td>step-62</td>
+ *       <td> Resonance frequency and bandgap of a phononic crystal. Elastic
+ *       wave equation with PMLs in the frequency domain. Parallelization via
```
Comment (bangerth) [inline doc/doxygen/tutorial/tutorial.h.in]:
```suggestion
 *       wave equation in the frequency domain with Perfectly Matched Layer boundary conditions. Parallelization via
```

**Comment by dangars** [comment]
I addressed the suggested changed

**Comment by dangars** [comment]
Do you mean that I add another Changelog entry for step-62? We already have #7994

**Comment by bangerth** [comment]
Ah, no. I just reviewed too many patches and forgot what I had already approved :-)

**Comment by dangars** [comment]
Ok :)

**Comment by bangerth** [comment]
Not the first time this happens to me...

**Pull request merged**